content/en/real_user_monitoring/browser/optimizing_performance.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Optimizing Performance
+aliases:
+  - real_user_monitoring/browser/monitoring_performance_vitals/
+further_reading:
+  - link: "https://learn.datadoghq.com/courses/core-web-vitals-lab"
+    tag: "Learning Center"
+    text: "Interactive Lab: Core Web Vitals"
+  - link: "https://www.datadoghq.com/blog/real-user-monitoring-with-datadog/"
+    tag: "Blog"
+    text: "Real User Monitoring"
+  - link: "https://www.datadoghq.com/blog/core-web-vitals-monitoring-datadog-rum-synthetics/"
+    tag: "Blog"
+    text: "Monitor Core Web Vitals with Datadog RUM and Synthetic Monitoring"
+---
+
+## Overview
+
+{{< img src="real_user_monitoring/browser/optimizing_performance/optimization-workflow.mp4" alt="RUM Performance Optimization helps you find the root cause of browser performance issues based on real user traffic." video="true" >}}
+
+The Optimization page helps identify the root cause of browser performance issues using real user traffic data. Here, you can troubleshoot browser metrics such as [Core Web Vitals][1] (CWV), [Loading Time][2] to determine the causes of slow pages, with all relevant information in one place.
+
+## Prerequisites
+
+To optimize your application effectively, ensure you are using:
```

Comment:
```suggestion
To optimize your application, ensure you are using:
```

Reply:
"To optimize" inherently implies doing something effectively. Optimization, by its very definition, means making something as effective or functional as possible.